### DISCUSSION TITLE
Collapsible sidebars for https://docs.modular.com/

### DISCUSSION THREAD

**Discussion by hirehamir** [modular/modular#2005] [Ideas]
https://docs.modular.com/mojo/manual/ is a bit less convenient to read in portrait mode because of the non-collapsible sidebars.

![image](https://github.com/modularml/mojo/assets/43425812/9e35c83d-842c-48c2-b645-ec5579e35220)

**Comment by iamtimdavis** [comment]
@scottamain can add some feedback here!

**Comment by scottamain** [comment]
Yeah, we can do that! 👍🏻
